engine/render/src/render/render_script.cpp

```diff
@@ -146,6 +259,13 @@ namespace dmRender
         {0, 0}
     };
 
+    static const luaL_reg RenderScriptConstantBufferArray_meta[] =
+    {
+        {"__index",     RenderScriptConstantBufferArray_index},
+        {"__newindex",  RenderScriptConstantBufferArray_newindex},
+        {0, 0}
+    };
```

Comment:
We don't need a gc for this since we don't allocate any memory for the table elements